testing/fzf/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=fzf
+_pkgver="0.17.0-2"
+pkgver=${_pkgver/-/.}
+pkgrel=0
+pkgdesc="A command-line fuzzy finder"
+url="https://github.com/junegunn/fzf"
+arch="all"
+license="MIT"
+makedepends="go"
+subpackages="$pkgname-tmux"
+source="$pkgname-$_pkgver.tar.gz::https://github.com/junegunn/fzf/archive/$_pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$_pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	make test
+}
+
+package() {
+	cd "$builddir"
+
+	# Just copies the target binary to $buildir/bin
+	make install
+
+	install -Dm0755 bin/fzf $pkgdir/usr/bin/fzf
+}
+
+tmux() {
+	depends="tmux bash"
+	makedepends="$depends"
```

Comment:
this will not work. the split function is not execute til after the package is built, so there is no way for abuild to be aware of those dependencies til after the package is built. We need to have it at the top of the build.

Reply:
I'm wondering, these are just run-time dependencies. They are never needed during building, and only for this sub-package. Is it than still necessary to mark them as global dependencies (defeating the purpose, only requiring bash and tmux if you want the tmux integration)